pkg/login/login_test.go

```diff
@@ -57,6 +57,13 @@ func TestProviders(t *testing.T) {
 				w.Header().Add("WWW-Authenticate", fixture.authChallenge)
 				w.WriteHeader(401)
 			})
+		} else {
+			mux.HandleFunc("/pkgpanda/active.buildinfo.full.json", func(w http.ResponseWriter, req *http.Request) {
+				assert.Equal(t, "HEAD", req.Method)
+				assert.Equal(t, "", req.Header.Get("Authorization"))
+
+				w.WriteHeader(200)
+			})
```

Comment:
It would make sense to specifically test for a `ErrAuthDisabled` error, otherwise the test would succeed even without your fix. Maybe that'd be simpler to have a separate test function for this case.

Reply:
I originally did that but figured it might make some sense to integrate into that function. If I back out those changes though what is https://github.com/dcos/dcos-cli/pull/1367/files#diff-3abfe1b27be64ad5227dbe318509dedeL22 supposed to be checking?

Would it be better to add another check
```go
if fixture.authChallenge == "" {
    require.Error(t, err, ErrAuthDisabled)
}
```?

Reply:
https://github.com/dcos/dcos-cli/pull/1367/files#diff-3abfe1b27be64ad5227dbe318509dedeL22 would be like testing against a non DC/OS cluster.

If you think that makes sense as part of this function that's fine too.